docker/docker-compose-local-devel.yml

```diff
@@ -0,0 +1,107 @@
+version: '3.8'
+
+services:
+
+  #------- Secrets
+  generator:
+    image: 792926601177.dkr.ecr.us-east-2.amazonaws.com/specular-platform:specular-latest
+    volumes:
+      - ../workspace:/specular/workspace
+      - ../sbin:/specular/sbin
+    working_dir: /specular/workspace
+    entrypoint: /bin/bash ../sbin/generate_secrets.sh -d -j -y
```

Comment:
[Not something to address] One thing that we should keep in mind for the future here is we'll have to implement an upgrade cycle, and we'll probably, when CLI tool is implemented (@macwis) want to re-use the generated secrets.